themes/custom/droopler_theme/js/main-menu.js

```diff
@@ -36,10 +36,16 @@
     attach: function (context, settings) {
       $ ('#navbar-main button.navbar-toggler', context).click(function() {
         // Avoids classes toggle while collapsing.
-        if((!$('body').hasClass('navbar-open') && !$('.navbar').hasClass('collapsing')) || ($('.navbar').hasClass('show'))) {
-          $('body').toggleClass('navbar-open', !$(this).is('[aria-expanded="true"]'));
-          $('.navbar').toggleClass('open', !$(this).is('[aria-expanded="true"]'));
+        var aria = $(this).attr('aria-expanded');
+        if ((!$('body').hasClass('navbar-open') && !$('.navbar').hasClass('collapsing'))) {
+          $('body').toggleClass('navbar-open');
+          $('.navbar').toggleClass('open');
         }
+        else {
+          $('body').toggleClass('navbar-open');
+          $('.navbar').toggleClass('open');
+        }
+        $(this).attr('aria-expanded', (aria === 'false'));
```

Comment:
why are you setting this attribute here, if it's already there ?
![image](https://user-images.githubusercontent.com/37533838/71666557-a7e37200-2d61-11ea-8729-a41fd24da4a5.png)